src/vasp.jl

```diff
@@ -165,4 +165,59 @@ end
 
 readDOSCAR(filename::AbstractString) = open(readDOSCAR, filename)
 
-readDOSCAR() = open(readDOSCAR, "DOSCAR")
\ No newline at end of file
+readDOSCAR() = open(readDOSCAR, "DOSCAR")
+
+function readPROCAR(io::IO)
+    ln = readline(io)
+    
+    if cmp(split(ln)[2], "lm") == -1
+        error("Not a lm-decomposed PROCAR. Use a different PROCAR.")
+    end
+    
+    # Get number of kpoints, number of bands, number of ions
+    ln = readline(io)
+    nkpt = parse(Int,split(ln)[4])
+    nband = parse(Int,split(ln)[8])
+    nion = parse(Int,split(ln)[12])
+    
+    # Initializes containers... to be fixed
+    kptlist = KPointList{3}(Vector{SVector{3,Float64}}(undef, nkpt))
+    # Contains kpoint weight. Unused for now.
+    # kptwt = Vector{Float64}(undef,nkpt)
+    bandatkpt = Vector{BandAtKPoint}(undef,nkpt)
+    energies = Matrix{Float64}(undef, nkpt, nband)
+    occupancies = Matrix{Float64}(undef, nkpt, nband)
+    projband = Array{Float64,4}(undef,9,nion,nband,nkpt)
```

Comment:
Fine for now, though I'd like to see if there's a way of dynamically selecting the array size depending on the types of orbitals present. (In case we ever work with lanthanides/actinides)